src/components/Button/ButtonContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { Button } from "./Button";
+import { ButtonContainerProps } from "./interfaces";
+
+export const ButtonContainer: React.FC<ButtonContainerProps> = ({
+  btnClass,
+  btnType,
+  onSub,
+  btnText,
+}) => {
+  const onSubmitForm = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    onSub();
+  };
+  return (
+    <Button
+      btnClass={btnClass}
+      btnType={btnType}
+      onSubmit={onSubmitForm}
+      btnText={btnText}
+    />
+  );
+};
```

Comment:
Use a simple button. Don't try to spread the logic. I don't see the reason to preventDefault. If you need to do it- it'll be easier to do it on some upper level